README.md

```diff
@@ -20,7 +20,7 @@ Website: https://fairlearn.org/
 ## Current release
 
 - The current stable release is available at
-  [Fairlearn v0.6.2](https://github.com/fairlearn/fairlearn/tree/release/v0.6.2).
+  [Fairlearn v0.7.0](https://github.com/fairlearn/fairlearn/tree/release/v0.7.0).
```

Comment:
here we could instead refer users to the `pypi.org` page which would show them the latest release by default, or our website which directs users to the latest release. And not change it here with every release.